packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -45,6 +45,14 @@ const multiSelectComboBox = css`
     flex-basis: 0;
     padding: 0;
   }
+
+  :host([all-chips-visible]) #chips {
+    display: contents;
+  }
+
+  :host([all-chips-visible]) [class$='container'] {
+    width: fit-content;
```

Comment:
This ensures the component expands when selecting a new item, until `max-width` is reached.